time/AddTaskViewController.swift

```diff
@@ -43,12 +34,11 @@ class AddTaskViewController: UIViewController, UITextFieldDelegate {
         // Dispose of any resources that can be recreated.
     }
     @IBAction func saveButton(_ sender: UIBarButtonItem) {
-        print("KKKKKK")
         let name = taskName.text ?? ""
                 let time = 12345667
-                let red = 123.0
-                let green = 223.0
-                let blue = 34.0
+                let red = drand48()
```

Comment:
I would prefer to use a fixed set of _attractive_ colors e.g: http://flatuicolors.com than to "_randomise_" them.